### ISSUE TITLE
`@stdlib/stats-pcorrtest`: missing pcorr key in pcorrtest Results type

### ISSUE THREAD

**Issue by andrefs** [stdlib-js/stdlib#1153]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@stdlib/stats-pcorrtest@0.1.0` for the project I'm working on.

The `pcorrtest` function returns an object. The examples in the README.md show that this object has a `pcorr` key with the value for the correlation coefficient. The implementation does indeed set the value for that key, but that key is missing from the `Results` type returned by `pcorrtest`.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@stdlib/stats-pcorrtest/docs/types/index.d.ts b/node_modules/@stdlib/stats-pcorrtest/docs/types/index.d.ts
index c3ff1e4..260a66d 100644
--- a/node_modules/@stdlib/stats-pcorrtest/docs/types/index.d.ts
+++ b/node_modules/@stdlib/stats-pcorrtest/docs/types/index.d.ts
@@ -76,6 +76,11 @@ interface Results {
 	*/
 	nullValue: number;
 
+	/**
+	 * Correlation coefficient.
+	 */
+	pcorr: number;
+
 	/**
 	* Alternative hypothesis (`two-sided`, `less` or `greater`).
 	*/
@@ -117,7 +122,7 @@ interface Results {
 * var y = [ 3, 2, 4, 1, 2, 4 ];
 * var out = pcorrTest( x, y );
 */
-declare function pcorrTest( x: NumericArray, y: NumericArray, options?: Options ): Results; // tslint-disable-line max-line-length
+declare function pcorrTest(x: NumericArray, y: NumericArray, options?: Options): Results; // tslint-disable-line max-line-length
 
 
 // EXPORTS //
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by kgryte** [comment]
@andrefs Thanks for filing this issue. Will circle back once we investigate.

**Comment by Planeshifter** [comment]
@andrefs We have meanwhile published `stats-pcorrtest@0.1.1`, which contains the corrected type definitions. Thanks again for filing this detailed issue description and patch; it's much appreciated! Please let us know should you encounter any further issues.

**Comment by andrefs** [comment]
Many thanks! :tada:
